plugins/templates-maven-plugin/src/main/java/com/google/cloud/teleport/plugin/maven/TemplatesStageMojo.java

```diff
@@ -1085,7 +1085,7 @@ private void stageYamlUsingDockerfile(
 
     File cloudbuildFile = File.createTempFile("cloudbuild", ".yaml");
     try (FileWriter writer = new FileWriter(cloudbuildFile)) {
-      String cacheFolder = imagePathTag.substring(0, imagePathTag.lastIndexOf('/')) + "/cache";
+      // String cacheFolder = imagePathTag.substring(0, imagePathTag.lastIndexOf('/')) + "/cache";
```

Comment:
actually can we add a TODO to re-enable cache after conflict resolved (can do after release cut, or as part of subsequent PRs, since running all tests are expensive)